package.json

```diff
@@ -1,22 +1,27 @@
 {
   "name": "@google-cloud/cloud-rad",
-  "version": "0.2.15",
+  "version": "0.2.19",
   "description": "",
   "main": "index.js",
   "repository": "googleapis/cloud-rad",
   "bin": {
     "cloud-rad": "./main.sh"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha"
   },
   "author": "Google Inc",
   "license": "Apache-2.0",
   "dependencies": {
-    "@microsoft/api-documenter": "https://gitpkg.now.sh/googleapis/rushstack/apps/api-documenter?main",
+    "@googleapis/api-documenter": "~7.13.0",
```

Comment:
Awesome, we're publishing the `api-documenter` now? If you need any help setting up automation for this reach out in the GitHub Automation channel.

Reply:
Yes, we are! I'll reach out so we can get that automated, thanks!